src/sentry/integrations/utils/code_mapping.py

```diff
@@ -19,7 +19,7 @@
 # Read this to learn about file extensions for different languages
 # https://github.com/github/linguist/blob/master/lib/linguist/languages.yml
 # We only care about the ones that would show up in stacktraces after symbolication
-EXTENSIONS = ["js", "jsx", "tsx", "ts", "mjs", "py", "rb", "rake", "php", "go"]
+EXTENSIONS = ["js", "jsx", "tsx", "ts", "mjs", "py", "rb", "rake", "php", "go", "cs"]
```

Comment:
Might consider `fs` too which is F# (a lot less common than C# but it's the same runtime, SDK etc. Paths are the same etc.